stable/katafygio/README.md

```diff
@@ -0,0 +1,81 @@
+# Katafygio
+
+[Katafygio](https://github.com/bpineau/katafygio) discovers Kubernetes objects (deployments, services, ...), and continuously saves them as YAML files in a Git repository. This provides real-time, continuous backups, and keeps detailed changes history.
+
+## TL;DR;
+
+```bash
+$ helm install stable/katafygio
+```
+
+## Introduction
+
+This chart installs a [Katafygio](https://github.com/bpineau/katafygio) deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+## Prerequisites
+
+- Kubernetes 1.8+
```

Comment:
We officially support the current and previous minor versions. Can we go with 1.9+?